### DISCUSSION TITLE
入口组件，function component 的useEffect 是否可以替代 class component 的 onLaunch

### DISCUSSION THREAD

**Discussion by Yang03** [NervJS/taro#12354] [Q & A] [answered]
### 这个特性解决了什么问题？
希望在小程序onLaunch 的时候，加载一下小程序的初始数据

### 这个 API 长什么样？
function component 入口有onLaunch 相同的hook

**Comment by TheKonka** [comment, marked as answer]
3.5 版本 useLaunch  
3.5之前的版本 useDidShow
